### ISSUE TITLE
SkyCoord objects with non-scalar attributes broken in 1.3.1

### ISSUE THREAD

**Issue by StuartLittlefair** [astropy/astropy#5884]
Coordinates with non-scalar attributes basically do not work, because the is_equivalent_frame routine is broken in 1.3.1. For example

```
times = Time(["2015-08-28 03:30", "2015-08-28 12:00", "2015-09-05 10:30", "2015-09-15 18:35"])
sun = coord.get_sun(times)
moon = coord.get_moon(times)
elongation = sun.separation(moon)
```

Note that @mhvk has actually already fixed this in #5751, but that merged PR was milestoned for 2.0 by @eteq I believe. What is needed is simply to replace 

```
if getattr(self, fattrnm) != getattr(other, fattrnm):
```

with 

```
if np.any(getattr(self, fattrnm) != getattr(other, fattrnm))
```

**Comment by StuartLittlefair** [comment]
Whilst this is a pretty trivial fix, I'm a bit confused as to how to go about it, since the necessary change is already in the master branch!

**Comment by bsipocz** [comment]
@StuartLittlefair - you can open a PR directly against the v1.3.x branch rather than master. And I don't mind to have a quick 1.3.2 release if the subpackage maintainers decide this is critical enough to get out soon.

**Comment by StuartLittlefair** [comment]
Thanks!

**Comment by bsipocz** [comment]
Fixed by #5887 

(weird that it didn't get auto closed...)
